core/src/subgraph/instance_manager.rs

```diff
@@ -379,9 +397,8 @@ fn process_block<B, S, T>(
     logger: Logger,
     ctx: IndexingContext<B, S, T>,
     block_stream_cancel_handle: CancelHandle,
-    needs_restart: Arc<AtomicBool>,
     block: EthereumBlock,
-) -> impl Future<Item = IndexingContext<B, S, T>, Error = CancelableError<Error>>
+) -> impl Future<Item = (IndexingContext<B, S, T>, bool), Error = CancelableError<Error>>
```

Comment:
Can we document the meaning of the bool above the function?

Reply:
Done